lib/node_modules/@stdlib/repl/lib/syntax_highlighter.js

```diff
@@ -115,28 +115,35 @@ setNonEnumerableReadOnly( SyntaxHighlighter.prototype, '_highlightLine', functio
 	var highlightedLine = '';
 	var resetCode = ANSI[ 'reset' ];
 	var colorCode;
-	var colors = this._themes[ this._theme ];
 	var offset = 0;
+	var color;
+	var theme = this._themes[ this._theme ];
 	var token;
 	var i;
+	var j;
 
 	// Sort and traverse the tokens...
 	tokens.sort( tokenComparator );
 	for ( i = 0; i < tokens.length; i++ ) {
 		token = tokens[ i ];
-		colorCode = ANSI[ colors[ token.type ] ];
-
-		// Highlight token if it's color exists in the theme...
-		if ( colorCode ) {
-			highlightedLine += line.slice( offset, token.start ); // add text before token
-			highlightedLine += colorCode; // insert colorCode
-			highlightedLine += line.slice( token.start, token.end ); // add token
-			highlightedLine += resetCode; // reset color
-			offset = token.end;
+		color = theme[ token.type ];
+		if ( !color ) {
+			continue; // no color defined for the token type
 		}
+		color = color.split( ' ' ); // allow combination of styles. eg: `bold magenta`
+		highlightedLine += line.slice( offset, token.start ); // add text before token
+		for ( j = 0; j < color.length; j++ ) {
```

Comment:
@Snehil-Shah Just checking. Does it matter whether `bold magenta` or `magenta bold`? Meaning, does order matter?

Reply:
@kgryte Order doesn't matter; works both ways.